### ISSUE TITLE
vue/html-indent indent inside vue tag

### ISSUE THREAD

**Issue by rshaibakov** [vuejs/eslint-plugin-vue#361]
**Tell us about your environment**
Mac OS X 10.13.2
Sublime Text 3

* **ESLint Version: 4.14.0** 
* **eslint-plugin-vue Version: 4.2.0**
* **Node Version: 8.9.4**

**Please show your full configuration:**

```
module.exports = {
  env: {
    browser: true,
    es6: true
  },
  extends: [
    'standard',
    'plugin:vue/recommended'
  ],
  plugins: ['vue'],
  parserOptions: {
    parser: 'babel-eslint',
    ecmaVersion: 8,
    sourceType: 'module'
  },
  rules: {
    'vue/no-parsing-error': ['error', {
      'x-invalid-end-tag': false
    }],
    'vue/html-indent': ['error', 2, {
       attribute: 1,
       closeBracket: 0,
       ignores: [],
       alignAttributesVertically: true
    }]
  },
  globals: {}
};


```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
  <header>
    <Row justify="center">
      <Col>
        <router-link to="/home/">
          <img
            src="/static/logo.png"
            alt="logo">
        </router-link>
      </Col>
    </Row>
  </header>
</template>

```

**What did you expect to happen?**

```vue

<template>
  <header>
    <Row justify="center">
      <Col>
        <router-link to="/home/">
          <img
            src="/static/logo.png"
            alt="logo">
        </router-link>
      </Col>
    </Row>
  </header>
</template>

```

**What actually happened? Please include the actual, raw output from ESLint.**

Eslint swears on indent inside tag Col. But there indented everything is fine. He wants to router-link was with tag Col. How wrong is that?

   5:1  error  Expected indentation of 6 spaces but found 8 spaces    vue/html-indent
   6:1  error  Expected indentation of 8 spaces but found 10 spaces   vue/html-indent
   7:1  error  Expected indentation of 10 spaces but found 12 spaces  vue/html-indent
   8:1  error  Expected indentation of 10 spaces but found 12 spaces  vue/html-indent
   9:1  error  Expected indentation of 6 spaces but found 8 spaces    vue/html-indent
  12:1  error  Expected indentation of 6 spaces but found 8 spaces    vue/html-indent

**Comment by mysticatea** [comment]
Thank you for the report.

Hmm, `<col>` is a void HTML element (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/col), so the rule says that the `router-link` is not a child of the `<col>` element.

**Comment by rshaibakov** [comment]
Oh yeah. It's my fault. I forgot about the html element col and called a component of vue as well. Sorry! Thank you so quickly answered!
